plugins/assimp/module/vtkF3DAssimpImporter.cxx

```diff
@@ -293,14 +308,10 @@ class vtkF3DAssimpImporter::vtkInternals
         case aiShadingMode_Minnaert:
           property->SetInterpolationToPhong();
           break;
-        case aiShadingMode_OrenNayar:
-        case aiShadingMode_CookTorrance:
-        case aiShadingMode_Fresnel:
-          property->SetInterpolationToPBR();
-          break;
```

Comment:
why removing this ?

Reply:
In the case of 3MF, no shading method is specified, so we rely on the default shading of VTK (Phong).  
However I encountered some materials with the diffuse color to black with result in a completely black model. With PBR, even if the model is black, we can still have nice reflections.

Reply:
I reverted this in my last commit, it was causing an issue with FBX.